src/http.cc

```diff
@@ -207,6 +207,9 @@ minio::http::Response minio::http::Request::execute() {
   std::istream body_stream(&charbuf);
 
   switch (method) {
+    case Method::kDelete:
+    case Method::kGet:
+      break;
```

Comment:
Is it rneeded for `macos`?

Reply:
its a `clang` complaint @balamurugana to make sure to all `enums` are present in the `switch`